wgpu-core/src/lib.rs

```diff
@@ -3,8 +3,36 @@
 //! into other language-specific user-friendly libraries.
 //!
 //! ## Feature flags
-#![doc = document_features::document_features!()]
+// NOTE: the above should be kept in sync. with `Cargo.toml`!
```

Comment:
Should this be "the below"?

Reply:
oderp, yes, that should be "below". 😅

Fixed with 3ebec9c13; I amended "the above" to be "feature docs. below".